ct-app/postman/postman_tasks.py

```diff
@@ -164,7 +165,7 @@ async def async_send_1_hop_message(
         api, peer_id, max_possible, node_peer_id, timestamp, params.param.batch_size
     )
 
-    status = TaskStatus.SUCCESS if relayed == expected_count else TaskStatus.SPLITTED
+    status = TaskStatus.SPLITTED if relayed < expected_count else TaskStatus.SUCCESS
```

Comment:
Although the proper state should be `TaskStatus.SPLIT`

Reply:
True, and fixed,